chipsec/utilcmd/decode_cmd.py

```diff
@@ -58,28 +58,29 @@
 
 class DecodeCommand(BaseCommand):
 
-    def requires_driver(self):
+    def requires_driver(self) -> bool:
         parser = ArgumentParser(usage=__doc__)
         parser.add_argument('_rom', metavar='<rom>', help='file to decode')
         parser.add_argument('_fwtype', metavar='fw_type', nargs='?', help='firmware type', default=None)
         parser.parse_args(self.argv[2:], namespace=self)
         return False
 
-    def decode_types(self):
-        self.logger.log("\n<fw_type> should be in [ {} ]\n".format(" | ".join(["{}".format(t) for t in uefi_platform.fw_types])))
+    def decode_types(self) -> None:
+        self.logger.log(f'\n<fw_type> should be in [ {" | ".join([f"{t}" for t in uefi_platform.fw_types])} ]\n')
 
-    def decode_rom(self):
-        self.logger.log("[CHIPSEC] Decoding SPI ROM image from a file '{}'".format(self._rom))
+    def decode_rom(self) -> bool:
+        self.logger.log(f'[CHIPSEC] Decoding SPI ROM image from a file \'{self._rom}\'')
         f = read_file(self._rom)
         if not f:
             return False
         (fd_off, fd) = get_spi_flash_descriptor(f)
         if (-1 == fd_off) or (fd is None):
-            self.logger.log_error("Could not find SPI Flash descriptor in the binary '{}'".format(self._rom))
+            self.logger.log_error(f'Could not find SPI Flash descriptor in the binary \'{self._rom}\'')
             self.logger.log_information("To decode an image without a flash decriptor try chipsec_util uefi decode")
             return False
+        return True
```

Comment:
This `return True` needs to be moved a few lines down, where the function ends (line 118)